rest-server-driver/pom.xml

```diff
@@ -1,120 +1,125 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-
-	<artifactId>rest-server-driver</artifactId>
-	<name>REST Server Driver</name>
-	<description>Test Driver to test your RESTful clients</description>
-
-	<parent>
-		<groupId>com.github.rest-driver</groupId>
-		<artifactId>rest-driver</artifactId>
-		<version>2.0.1-SNAPSHOT</version>
-	</parent>
-
-	<dependencies>
-
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>rest-client-driver</artifactId>
-            <version>${project.version}</version>
-            <scope>test</scope>
-        </dependency>
+  <modelVersion>4.0.0</modelVersion>
```

Comment:
Remove whitespace changes

Reply:
Sure. I'll try to format them with 4 spaces. I wanted to tidy it up, because some of the lines were unaligned.

Reply:
I think if we want a consistent style then we should add some IntelliJ formatting xml, one for Eclipse, and then also add in checkstyle. I can add an issue to support that